README.md

```diff
@@ -1,2 +1,2 @@
 # wavs-wasi
-wasi-utils and wit files for easy MIT import
+wasi-utils and wit files for easy WIT import
```

Comment:
I think this is meant to be MIT given the license

Reply:
D'oh! Great catch.. will change back

Reply:
Done in https://github.com/Lay3rLabs/wavs-wasi/pull/14/commits/e41dffd20d235e13818c55445c6248dca5e234be